nautobot/core/templates/nautobot_config.py.j2

```diff
@@ -104,26 +113,26 @@ ALLOWED_URL_SCHEMES = (
 
 # Optionally display a persistent banner at the top and/or bottom of every page. HTML is allowed. To display the same
 # content in both banners, define BANNER_TOP and set BANNER_BOTTOM = BANNER_TOP.
-BANNER_TOP = ""
-BANNER_BOTTOM = ""
+BANNER_TOP = os.getenv("NAUTOBOT_BANNER_TOP", "")
+BANNER_BOTTOM = os.getenv("NAUTOBOT_BANNER_BOTTOM", "")
 
 # Text to include on the login page above the login form. HTML is allowed.
-BANNER_LOGIN = ""
+BANNER_LOGIN = os.getenv("NAUTOBOT_BANNER_LOGIN", "")
 
-# Cache timeout in seconds. Cannot be 0. Defaults to 900 (15 minutes). To disable caching, set CACHEOPS_ENABLED to False
-CACHEOPS_DEFAULTS = {"timeout": 900}
+# Cache timeout in seconds. Cannot be 0. Defaults to 300 (5 minutes). To disable caching, set CACHEOPS_ENABLED to False
+CACHEOPS_DEFAULTS = {"timeout": int(os.getenv("NAUTOBOT_REDIS_TIMEOUT", 300))}
```

Comment:
Maybe asked already, but why change this from 900 to 300?

Reply:
we used to only have 1 timeout that was 300, should probably have 2 variables, fixing